src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -500,8 +500,9 @@
        :entityType "Task"})
     (range (rand-int 50)))})
 
-(defn get-configuration [namespace name snapshot-id]
-  {:path (str "/configurations/" namespace "/" name "/" snapshot-id)
+(defn get-configuration [namespace name snapshot-id & [payload-as-object?]]
+  {:path (str "/configurations/" namespace "/" name "/" snapshot-id
+              (when payload-as-object? "?payloadAsObject=true"))
```

Comment:
not for this PR; for some future: we should probably be using some kind of Uri library that allows adding/removing query params, instead of concatenating strings ourselves.